modules/nf-scil/betcrop/synthbet/meta.yml

```
@@ -0,0 +1,53 @@
+---
+name: "betcrop_synthbet"
+description: Perform brain extraction using synthstrip on image
+keywords:
+  - anatomical image
+  - BET
+  - freesurfer
+tools:
+  - "Freesurfer":
+      description: "Software package for the analysis and visualization of structural and functional neuroimaging data."
+      homepage: "https://surfer.nmr.mgh.harvard.edu/fswiki"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'test', single_end:false ]`
+
+  - image:
+      type: file
+      description: Nifti image volume to perform BET.
+      pattern: "*.{nii,nii.gz}"
+
+  - weights:
+      type: file
+      description: Alternative model weights
+      pattern: "*.{nii.nii.gz}"
```

Alternative model weights are in a pytorch model file. 
```suggestion
      pattern: "*.pt"
```